.github/workflows/rebuild-100k.yml

```diff
@@ -0,0 +1,57 @@
+name: Rebuild 100k sample
+
+on:
+  # cron job once a week on mondays
+  schedule:
+    - cron:  '* * * * 1'
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  gisaid:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: nextstrain/.github/actions/setup-nextstrain-cli@master
+      with:
+        python-version: 3.9
+
+    - name: Launch build
+      run: |
+        set -x
+
+        declare -a config
+        config+=(slack_token=$SLACK_TOKEN)
+
+        nextstrain build \
+          --aws-batch \
+          --detach \
+          --cpus 16 \
+          --memory 14GiB \
```

Comment:
Don't we automatically get a lot more memory when we specify 16 CPUs?

Could future proof by allowing as much memory as we get for the CPUs

Reply:
You're right. Our [nextstrain-job queue](https://us-east-1.console.aws.amazon.com/batch/home?region=us-east-1#queues/detail/arn:aws:batch:us-east-1:827581582529:job-queue/nextstrain-job-queue) uses the [c5-instances-2023-01-17](https://us-east-1.console.aws.amazon.com/batch/home?region=us-east-1#compute-environments/ec2/detail/arn:aws:batch:us-east-1:827581582529:compute-environment/c5-instances-2023-01-17) compute environment which uses [c5 instance types](https://aws.amazon.com/ec2/instance-types/). So 16 CPUs will get us 32GiB memory. I've updated this line accordingly.

P.S. This compute environment specifies both `"allocationStrategy": "BEST_FIT",` and `"desiredvCpus": 192,` so I hope deploying a job as we're doing here will still get a 16 CPU model? cc @tsibley 

P.P.S. Given 16 CPUs + 32GiB memory, how does the scheduler choose between c5.4xlarge and c5d.4xlarge?

Reply:
Running a test build of this PR (via `nextstrain build` as a new GitHub action isn't runnable from a PR) with `--cpus 16 --memory 31GiB` was allocated a machine with 16 vCPUs. It's not clear which actual model it uses.

Reply:
> P.S. This compute environment specifies both `"allocationStrategy": "BEST_FIT",` and `"desiredvCpus": 192,` so I hope deploying a job as we're doing here will still get a 16 CPU model? cc @tsibley

Yes. `desiredvCpus` is a gauge value that floats up and down with demand (between `minvCpus`, which we should always have at 0, and `maxvCpus`). Batch modifies it as necessary for jobs, and compute resources in other services (e.g. ECS in our case) scale up and down in response. You typically don't modify it directly.

> P.P.S. Given 16 CPUs + 32GiB memory, how does the scheduler choose between c5.4xlarge and c5d.4xlarge?

c5d is a separate instance family than c5, so those instances are not made available in our default `nextstrain-job-queue`.